assets/js/components/App.tsx

```diff
@@ -8,6 +8,7 @@ const OutfrontTakeoverTool = React.lazy(
 const Dashboard = React.lazy(() => import("./Dashboard/Dashboard"));
 const PlacesPage = React.lazy(() => import("./Dashboard/PlacesPage"));
 const AlertsPage = React.lazy(() => import("./Dashboard/AlertsPage"));
+const AlertDetails = React.lazy(() => import("./Dashboard/AlertDetails"));
```

Comment:
I haven't seen this approach before. So is the aim to improve performance by not rendering all of the components at the initial page load and only render them when a use select to go to either page?

Reply:
This is actually the way we accomplished splitting the code between the new Screenplay and the old OFT. The bootstrap we use in Screenplay messes up the styling in OFT. If we lazy load the components, they only render when visited, so the CSS does not load and apply on OFT components. In the future, this will go away and we will render the components normally.